### PR TITLE
Maintance

### PR DESCRIPTION
**Pull request by AarushThapa**
1. Custom route/ navigation
2. Structured

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...d/leo-rotract/src/components/Meeting/PreMeetingCard.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/src/components/MeetingDetails/Minute.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/src/components/MeetingDetails/MinuteDiscussion.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/src/components/MeetingDetails/MinuteMember.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/src/components/MeetingDetails/MinuteProfile.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/src/index.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/src/routes/Primary.js",[],"/media/aarush/New Volume/leo rotract frontend/leo-rotract/webpack.config.js",[],{"ruleId":"104","severity":2,"message":"105","line":6,"column":1,"nodeType":"106","messageId":"107","endLine":18,"endColumn":2},{"ruleId":"108","severity":2,"message":"109","line":32,"column":15,"nodeType":"110","messageId":"111","endLine":32,"endColumn":20},"react/prefer-stateless-function","Component should be written as a pure function","ClassDeclaration","componentShouldBePure","no-unused-vars","'props' is defined but never used.","Identifier","unusedVar"]
```
Comment (poudyalanil) [inline .eslintcache]: ?????

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+import PrimaryRoute from './routes/Primary'
 
-const App = () => {
-	console.log('one')
+const App = () => (
+	<Router>
+		<Switch>
+			<Route path="/" exact component={Login} />
+			<Route path="/minute" component={Minute} />
+			<Route path="/page-not-found" component={PageNotFound} />
```
Comment (poudyalanil) [inline src/App.js:16]: does this require separation?

File: src/components/404/index.js [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import styles from './style.module.css'
+
+class NotFoundPage extends React.Component {
+	render() {
+		return (
+			<div>
+				<p style={{ textAlign: 'center' }}>
```
Comment (poudyalanil) [inline src/components/404/index.js]: No inlines!

File: src/components/Club/Member.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+				</Grid>
+				<Grid item xs={10}>
+					<Typography variant="body1" component="div">
+						<Box pl={2}>{props.name}</Box>
+					</Typography>
+
+					<Typography variant="h5" component="h5">
+						<Box
+							fontWeight="fontWeightBold"
+							// pt={1}
```
Comment (poudyalanil) [inline src/components/Club/Member.js]: comment?

File: src/components/Dashboard/Dashboard.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { Typography, Box, Grid, Button, Paper, Avatar } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { People, Message } from '@material-ui/icons'
+import Meeting from '../Meeting/Meeting'
```
Comment (poudyalanil) [inline src/components/Dashboard/Dashboard.js:5]: directory??

File: src/components/Dashboard/Dashboard.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+import IconDescription from './IconDesciption'
+import Club from '../Club/Club'
```
Comment (poudyalanil) [inline src/components/Dashboard/Dashboard.js:7]: ?

File: src/components/Dashboard/NumberTitle.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const numTitle = makeStyles({
+	box: {
+		boxShadow: '0 -5px 0 0 rgba(191, 109, 36, 1), 0 0 0 0 rgba(0, 0, 0, 0.1)',
```
Comment (poudyalanil) [inline src/components/Dashboard/NumberTitle.js:8]: variables could have been nice

File: src/components/Dashboard/NumberTitle.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+		width: '80px',
```
Comment (poudyalanil) [inline src/components/Dashboard/NumberTitle.js]: em, rem, vh, vw >>>> px

File: src/index.html [hunk 9]
```diff
@@ -14,7 +14,7 @@
 				font-family: 'Poppins', sans-serif;
 				margin: 0;
 				padding: 0;
-				background-color: #fcead4;
+				/* background-color: #fcead4; */
```
Comment (poudyalanil) [inline src/index.html]: !comment

File: src/index.js [hunk 10]
```diff
@@ -1,8 +1,10 @@
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (poudyalanil) [inline src/index.js]: !comment

File: src/routes/Primary.js [hunk 11]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (poudyalanil) [inline src/routes/Primary.js]: !comment

**Review by poudyalanil — CHANGES_REQUESTED** [review]
Great 
Consider changes and PR again

**Comment by poudyalanil** [comment]
@AarushThapa 
Please rebase to master branch and try to squash all the commits

File: .eslintcache (hunk 1 above)
Comment (AarushThapa) [reply .eslintcache]: outdated

File: src/App.js (hunk 2 above)
Comment (AarushThapa) [reply src/App.js:16]: Clarify

File: src/components/Club/Member.js (hunk 4 above)
Comment (AarushThapa) [reply src/components/Club/Member.js]: done

File: src/components/Dashboard/Dashboard.js (hunk 5 above)
Comment (AarushThapa) [reply src/components/Dashboard/Dashboard.js:5]: clarify

File: src/components/Dashboard/NumberTitle.js (hunk 8 above)
Comment (AarushThapa) [reply src/components/Dashboard/NumberTitle.js]: em used

File: src/routes/Primary.js (hunk 11 above)
Comment (iammdsameer) [reply src/routes/Primary.js]: that is not a comment, instead is a type of special syntax that informs `eslint` to disable a default behaviour.

File: src/index.js (hunk 10 above)
Comment (AarushThapa) [reply src/index.js]: outdated

Comment (iammdsameer) [reply src/index.js]: that is not a comment, instead is a type of special syntax that informs `eslint` to disable a default behaviour.

File: src/routes/Primary.js (hunk 11 above)
Comment (AarushThapa) [reply src/routes/Primary.js]: done

File: src/index.html (hunk 9 above)
Comment (AarushThapa) [reply src/index.html]: fixed

**Review by iammdsameer — APPROVED** [review]

**Comment by iammdsameer** [comment]
@poudyalanil squash merge & delete the branch once reviewed.

File: src/index.js (hunk 10 above)
Comment (poudyalanil) [reply src/index.js]: Great

**Review by poudyalanil — APPROVED** [review]

**Pull request merged**